Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -3614,6 +3614,17 @@ - (void)attachmentButtonPressed
     [chooseMediaAction setValue:chooseMediaImage forKey:@"image"];
     [actionSheetController addAction:chooseMediaAction];
 
+    UIAlertAction *gifAction = [UIAlertAction
```

Comment:
Myles re-ordered the items in this action sheet.

Reply:
👍